lib/Trash/CollectivePageTrashItem.php

```diff
@@ -19,7 +19,7 @@ public function __construct(
 		IUser $user,
 		private string $mountPoint
 	) {
-		parent::__construct($backend, $originalLocation, $deletedTime, $trashPath, $fileInfo, $user);
+		parent::__construct($backend, $originalLocation, $deletedTime, $trashPath, $fileInfo, $user, null);
```

Comment:
@grnd-alt Maybe we can still split this, it just came to my mind that we'd probably want to have a version comparison here to only apply this for Nextcloud >= 30